src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -11,30 +11,30 @@ import kotlin.reflect.KClass
 
 class CalculatorTest : BehaviorSpec({
     // System Under Test 라는 의미이며 테스트 대상을 의미한다
-    val sut = Calculator()
+    val target = Calculator()
```

Comment:
```suggestion
    val systemUnderTest = Calculator()
```
주석에 명시하셨던 의미를 변수명으로 작성해보면 어떨까요?
어색하시다면, target 보다는 calculator 로 작성하셔도 무방합니다. 😄